<ui version="4.0" >
 <author>RaphaÃ«l Doursenaud</author>
 <class>configuration</class>
 <widget class="QWidget" name="configuration" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configuration</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>20</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Number of CD Writers</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>130</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Images Directory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton" >
   <property name="geometry" >
    <rect>
     <x>460</x>
     <y>130</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text" >
    <string>Change</string>
   </property>
   <property name="shortcut" >
    <string>Alt+C</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="image" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>170</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox" >
   <property name="geometry" >
    <rect>
     <x>470</x>
     <y>20</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>310</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
   <property name="shortcut" >
    <string>Alt+O</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="9" />
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>true</bool>
  </property>
  <property name="gridSnapY" >
   <bool>true</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
